doc/man5/fips_config.pod

```diff
@@ -107,6 +107,10 @@ See L<config(5)>.
 L<config(5)>
 L<openssl-fipsinstall(1)>
 
+=head1 HISTORY
+
+This functionality was added to OpenSSL 3.0.
```

Comment:
We typically write _added in_ rather than _added to_.
There are exceptions but they are, by far, the minority.

Reply:
I block copied it from another file.. Updated them all.